### ISSUE TITLE
"Defend Spera Anatrusk" is too difficult in a single ship

### ISSUE THREAD

**Issue by Pointedstick** [endless-sky/endless-sky#2054]
In the "Defend Spera Anatrusk" mission, the Kor Mereti fleet that warps in to attack Spera Anatrusk is simply too large and powerful.

I'm flying a lone Derecho, with no fleet. It's armed with four Dual Sunbeam Turrets. It's not a piddly little ship. No matter how many times I fight this battle, the Kor Mereti always prevail. Their Disruptors are just ruinously good, and the Wanderer ships often get disabled at 30-50% shields. This is not helped by the fact that Flycatchers attempt to board and repair disabled Wanderer ships, but fail (they just flit around uselessly). I had to cheat to win the battle, giving my ship 4 more turrets and half a dozen flamethrowers and immediately disabling the Model 512 and 256s.

I try not to fly an Archon or Arfecta or anything, and just play the game the like I hope a normal space captain might. I don't see a way for my ship to possibly turn the tide of this battle. If that's the point, and the game is telling me that I should really be flying around in a fleet of flamethrower Bactrians or something, well, since this mission ejects you from the planet and immediately into the battle, it's a little late for that...

**Comment by Amazinite** [comment]
>I'm flying a lone Derecho, with no fleet.

I think I've found your problem, sir. 

I recall talk of the FW campaign being designed to be possible (if somewhat hard) with a single human medium or heavy warship, but is this design philosophy necessarily suppose to transfer over to the Wanderer story? Just given some of the battles in the newest missions, I wouldn't say that making it possible with one ship was a focus. 

Personally I went through the newer missions with 3 ships—my custom fit Derecho and two unmoddified Kar Ik Vots—and it didn't seem particularly hard (until I lost my Kar Ik Vots at one point because I wasn't paying attention to their health and had to go buy some extra Derechoes to replace them, but even after that things were fine).

**Comment by manitscold123** [comment]
The missions are hard because this is the tier 3 storyline ( I think ) and the missions will progress to get harder. I just run around with a gigantic fleet of Automata at my back 

ignore the Quark Wardragons. Got them by killing some Quarg.

![screen shot 2017-01-22 at 7 31 36 pm](https://cloud.githubusercontent.com/assets/20450688/22190643/82a63770-e0d9-11e6-89f0-aa367563fd07.png)

If getting Korath Automata is too hard ( the self-destruct pisses me off ) Just get more Derechos ( I assume you have a great deal of credits ) or Tempests, Shield Beetles, and, if you're getting frisky, take out the two Pug Arfectas Hovering around Ap'arak ( they can be swarmed easily, and are t3, so really badass). Also, I am not sure how strong Coalition ships are, but they seem strong. Finishing their storyline will unlock their Shipyard, which is,  just do the jobs. The Wanderers will not lose

**Comment by Amazinite** [comment]
It's T2, not T3. And he's using a T2 ship, so that's why the question comes up. See my post for why.

**Comment by Pointedstick** [comment]
I did not find the FW campaign at all challenging with a single ship. And I easily beat the first half of the Hai/Wanderer storyline in a single copy of the most powerful Wanderer ship they had available at the time, always upgrading as it became available. I beat the Alpha carrier with my single ship. It seems odd that this would very suddenly no longer work in the middle of the Wanderer storyline.

I don't want to have to farm the Quarg or Kor automata or steal some Arfectas in order to beat the missions. This seems like rank powergaming to me and breaks me out of the game world.

I suppose I could build up a fleet of Derechos, but that would get really expensive, and I only have one cloaking device so the rest of my fleet couldn't cloak. I'm not aware of non-cheating, non-alien-farming methods of getting more cloaking devices, so that's out. And I prefer not to cheat to make money or fly around with a silly fleet of 1,000 Star Barges or something, so no, unlike some 😉 I do not have eleventy trillion credits in the bank to buy a huge warfleet at the drop of a hat.

I'd worried that eventually this game would stop supporting my preferred play style of having a single ship and not having to mess around with a fleet, and I hope the time hasn't come.

**Comment by manitscold123** [comment]
Like I said before, if you prefer going solo ( I do too ), you might want to get something better than a t2 derecho, b/c they are much weaker than Korath Automata, so maybe find a solo wardragon, or like I said, the 2 bloody Arfectas in Wanderer Space. I'm sure that there are better ships somewhere, but it can't be that bad.

**Comment by Pointedstick** [comment]
I would love to get a better solo ship than a Derecho. But I want to do it without cheating or attacking/farming the Quarg or Pug (breaks me out of the game world).

For a long time I tried to get a Kar Ik Vot, observing huge automata battles in my perma-cloaked Derecho (thanks to 6 Wanderer Ramscoopos), but they self-destructed every single time and eventually I got frustrated and gave up. I've got a job and a wife and two kids and don't have the time to grind video games the way I did when I was a teenager. I want them to be fun and respectful of my limited time.

**Comment by manitscold123** [comment]
Or you could hack it in. ( put it in sales for some wanderer world )

**Comment by jafdy** [comment]
@manitscold123 Your making a string of suggestions that Pointed doesn't want. This is not a "Help me" thread so stop treating it like one.
@Pointedstick This may change when the Coalition storyline comes along and you can buy T2.5 warships but until then we could use some insight into the design philosophy behind how hard these mission are meant to be.

**Comment by KiLEdEnNis** [comment]
you said you had a single Derecho with 4 Dual Sunbeam Turrets, but the main weakness I noticed about them is them being quite ineffective vs the Automata: their hull damage is junk, and the Automata has quite a bit of hull power
and not only your ship has that; all the Wanderer ships have those Sunbeams; the entire fleet is having a very hard time destroying the hulls of the drones (even though the Mereti hulls < Sestor hulls), but even Electron Turrets do more hull damage
and a side note: the Thunderhead Launchers are more effective vs shields than hull as well

so in short I think one of the problems is that the Sunbeams have bad hull damage, which amplifies the other problem; the Mereti fleet being too big (I didn't notice, I have a save file so old that I have a Hurricane and 4 Derecho's all with 2 Antimatter Cores & 4 Skylances)

**Comment by Amazinite** [comment]
>It seems odd that this would very suddenly no longer work in the middle of the Wanderer storyline.

Well the big difference now is that the power of your opponents jumped. The Wanderer start was very T1, ~T1.5, but after entering the eye things become T2. You aren't fighting Hai or Navy Carriers anymore, you're fighting an advanced robot army designed to kill everything in sight, so it isn't surprising to me that things became more difficult for solo ship users.

**Comment by KiLEdEnNis** [comment]
even Quarg ships travel in packs though now I think of it

**Comment by Pointedstick** [comment]
> Or you could hack it in. ( put it in sales for some wanderer world )

Like I said,

> I would love to get a better solo ship than a Derecho. But I want to do it without **cheating** or attacking/farming the Quarg or Pug (breaks me out of the game world).

If I'm gonna cheat, why not just give my ship five trillion shields and hull?

**Comment by SonEasterZombie** [comment]
you could try outfitting your ship with mereti weapons (since the sunbeams suck vs hull) or if it doesn't break the game for you try to disable a punisher

**Comment by onnikyla** [comment]
I agree with Pointedstick, The wanderer story is too hard. Completing it with a single ship (one you are supposed to be abled to get) is impossible. I had to use 3 hurricanes and 4 derechos to complete it, but it still took a lot of reloading.

**Comment by KiLEdEnNis** [comment]
@SonEasterZombie disabling a Punisher will give him negative rep to the Heliarch which would ruin future stuff so that's not an option

**Comment by Bladewood** [comment]
@KiLEdEnNis [WRONG.](http://steamcommunity.com/app/404410/discussions/0/143388408896167130/?tscn=1485277236#c143388408896925551)

**Comment by KiLEdEnNis** [comment]
@Bladewood I meant once you disable one you'll have negative rep, meaning they'll be hostile

**Comment by Bladewood** [comment]
@KiLEdEnNis STILL WRONG. Using a trick with mission Quarg ships and reputation.

**Comment by KiLEdEnNis** [comment]
didn't know then, can ya explain?

**Comment by Pointedstick** [comment]
It's true, these missions are simply too difficult.

I finally managed to cap a Kar Ik vot and reluctantly left it as an escort. Both of my ships are outfitted with a mix of Wanderer and Korath weapons. We get annihilated in the "Defend Laki Nemparu" mission. The Mereti fleets are just too huge.

I guess I'll go farm the Kor Sestor a bit more... 😕

**Comment by KiLEdEnNis** [comment]
( ͡°, _ʖ ͡°,)
or farm some Quarg

**Comment by DingusShingleton** [comment]
How many times is pointed going to have to stiffarm the "farm the quarg" suggestion before it finally stays down ><

**Comment by Bladewood** [comment]
The difference in value between the Kor Mereti and the Wanderers in "Wanderers: First Mereti Attack" is about 236.187 million credits, in favor of the Kor Mereti.

The difference in value between the Wanderers from "Wanderers: Mind Escorts" plus the converted Kor Mereti versus the unconverted Kor Mereti from "Wanderers: Mind 6" is 350.387M, in favor of the unconverted.

That's a very, VERY big difference for both, much more than a single heavy warship could make up.

**Comment by KiLEdEnNis** [comment]
I'd say increase the Wanderer escorts a bit too: 2 Hurri's, 3 Derecho's 4 Temp's to invade a Mereti station? meh

**Comment by Pointedstick** [comment]
I don't *like* farming alien species--especially one I have no reason to be not hostile toward. I never liked World of Warcraft and other similar games because "farming" feels too much like work. I get enough work in the rest of my life. I want games to be fun. A game ceases to be fun when its storyline forces me into doing something (grinding and farming) that I don't find fun.

**Comment by KiLEdEnNis** [comment]
most MMORPG's become grinds & then boring
Endless Sky should enable players to always be able to do something (unless out of content for now); not making you grind to be able to finish a story

**Comment by Pointedstick** [comment]
Precisely!

**Comment by endless-sky** [comment]
All the Wanderer missions are designed and tested to be beatable with a
single ship. Tier 2 combats take a lot longer than Tier 1 (the ships don't
evaporate as quickly), which allows you to use a bit more strategy. One
good approach is to run a bunch of enemy ships on a wild goose chase while
the NPCs mop up the rest; another is to make sure you're always focusing
fire on the same ship that the NPCs are attacking.

**Comment by Pointedstick** [comment]
Which single ship, and with what loadout, out of curiosity?

**Comment by endless-sky** [comment]
I used a Derecho up until the Hurricane was available, then a Hurricane with:

```
		"Dual Sunbeam Turret" 5

		"Blue Sun Reactor"
		"White Sun Reactor"
		"Systems Core (Large)"
		"Wanderer Ramscoop" 4
		"Fuel Pod"
		"Large Heat Shunt"
		"Small Heat Shunt" 2
		"Cloaking Device"
		"Mass Expansion" 4

		"Thruster (Planetary Class)"
		"Steering (Planetary Class)"
		"Steering (Asteroid Class)"
		"Jump Drive"
```

As others have noted, sunbeams aren't great against hull armor, but they have the advantage that they're enough to overheat whatever ship you're attacking soon after its shields drop.

**Comment by KiLEdEnNis** [comment]
ooh MZ uses cloaking devices & Korath stuff

**Comment by onnikyla** [comment]
I couldnt beat the wanderer storyline with that  setup. (might be because my game was lagging alot)

**Comment by endless-sky** [comment]
It's possible that my AI changes (specifically, having the AI prioritize armed targets over unarmed ships like the flycatchers) made this harder than it was when I first tested it. When I get the chance I'll re-play the mission to check the difficulty.

**Comment by endless-sky** [comment]
I made the Mereti a bit weaker (2 fewer 16s and 1 fewer 32 - see e0249e4). At that strength, I was able to beat the mission with only a Tempest after a couple of tries.

**Comment by Pointedstick** [comment]
Excellent.

**Comment by w0ntoan** [comment]
I had difficulty, but it wasn't impossible in 0.95. After the defense fleet was wiped out, I had to kite the remaining 1x 512, 3x 32, and 1x 16. I flew around in large circles to drag the fastest ships in the bunch to the front, then flew through the center with slightly less than matched speed, blasting away at max range until they were temporarily disabled. Rinse and repeat.

**Comment by KiLEdEnNis** [comment]
focus a ship the defense fleet attacks, or try and lure a bunch away from the fight so the defense fleet can overwhelm their part

**Comment by w0ntoan** [comment]
```
	"Dual Sunbeam Turret" 5

		"Blue Sun Reactor"
		"White Sun Reactor"
		"Systems Core (Large)"
		"Wanderer Ramscoop" 4
		"Fuel Pod"
		"Large Heat Shunt"
		"Small Heat Shunt" 2
		"Cloaking Device"
		"Mass Expansion" 4

		"Thruster (Planetary Class)"
		"Steering (Planetary Class)"
		"Steering (Asteroid Class)"
		"Jump Drive"
```

Wow, this is great set up. I switched two small heat shunts and fuel pod for a flamethrower and wanderer heat sink. I was able to defend Farpoint from the sestor fleet, albeit with a lot of abuse of that return distance and continuous fleets coming in.

**Comment by retrue** [comment]
I just played this mission today. I beat it using the cloaking a lot, like a coward, running away and assisting disabled Wanderer ships. I attacked some ships and cloaked quickly. It cost me several deaths. I only had one ship, a very modified Bactrian but without Quarg tech.
Anyway, my reason to post here is the flycatchers. Like it is said in the first post, they fly around but they are not able to help disabled ships.
When my ship got disabled I called flycatchers that were flying around. I asked for help, they replied but they were unable to enable my ship. I don't know if it is a bug or it is that so small ships are not able to repair big ones, but if that is the case some message should warn the player. Something like "Sorry Sir, but we are unable to assist you".

**Comment by Bladewood** [comment]
They're not supposed to.
Them being hail-able is an issue of its own, I think.

**Comment by KiLEdEnNis** [comment]
"I'm sorry, but I'm an automated machine, I lack the physical means to aid in your ship repairs."

**Comment by retrue** [comment]
The problem is that they answer to the hail and they try to repair the ship, uselessly, when they could be attacking/defending.

**Comment by Pointedstick** [comment]
@retrue: can you open a separate issue for that.

With the latest tweaks, I think this mission is fine now.
